adops/adops-planning-guide.md

```diff
@@ -30,6 +29,18 @@ There are several decisions you need to make as you’re planning out your Prebi
 {: .alert.alert-success :}
 As you go through the steps, we recommend that you document all your decisions. Many of the decisions will need to be made only once when you first set up Prebid, so good documentation will help you and future users understand the decisions and why they were made.
 
+## AdOps Video Overview
+
+An overview of the process of planning a Prebid integration for ad operations.
+
+<div style="padding:56.25% 0 0 0;margin: 1rem 0;position:relative;"><iframe src="https://player.vimeo.com/video/891677441?badge=0&amp;autopause=0&amp;player_id=0&amp;app_id=58479" frameborder="0" allow="autoplay; fullscreen; picture-in-picture" style="position:absolute;top:0;left:0;width:100%;height:100%;" title="Prebid Ad Operations Planning"></iframe></div><script src="https://player.vimeo.com/api/player.js"></script>
+
```

Comment:
I'll create an include for that as well as we have now quite a few videos on the page and duplicate this everywhere.

#5089

Reply:
@muuki88 - the link isn't the same. Note the number after /video is different for each. Bu that's the only difference - Is there a way to pass a parameter to an include?